praktikumsplaner-frontend/frontend/tests/unit/rules.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { useRules } from "@/composables/rules";
+
+describe("rules test", () => {
+    const errorMessage = "Error";
+    it("tests fileFormatRules return true", () => {
+        const validationRules = useRules();
+
+        const txtRule = validationRules.fileFormatRule(
+            "text/plain",
+            errorMessage
+        );
+        const txtFile = new File(["foo"], "foo.txt", {
+            type: "text/plain",
+        });
+
+        expect(txtRule(txtFile)).toBe(true);
+    });
+    it("tests fileFormatRules returns Error when using wrong fileformat", () => {
+        const validationRules = useRules();
+
+        const txtRule = validationRules.fileFormatRule(
+            "audio/aac",
+            errorMessage
+        );
+        const txtFile = new File(["foo"], "foo.txt", {
+            type: "text/plain",
+        });
+
+        expect(txtRule(txtFile)).toBe(errorMessage);
+
+        const nullFile = null;
+        let undefinedFile;
+
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
+        expect(txtRule(nullFile)).toBe(errorMessage);
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+        // @ts-ignore
+        expect(txtRule(undefinedFile)).toBe(errorMessage);
```

Comment:
bitte jeweils in einen eigenen Testcase auslagern. Wenn du alle invalide-Cases gruppiert willst kann ein `describe` wieder zum Einsatz kommen.